core/src/main/java/de/mirkosertic/bytecoder/core/ir/ResolvedClass.java

```diff
@@ -109,11 +110,18 @@ public void registerDirectSubclass(final ResolvedClass cl) {
     public ResolvedMethod resolveMethod(final String methodName, final Type methodType, final AnalysisStack analysisStack) {
         final ResolvedMethod m = resolveMethodInternal(methodName, methodType, analysisStack, false);
         if (m == null) {
-            throw new IllegalStateException("No such method : " + classNode.name + "." + methodName + methodType);
+            throw new IllegalStateException("No such method : " + classNode.name + "." + methodName + methodType + printStackTrace(analysisStack));
```

Comment:
Maybe we can wrap this Exception in an de.mirkosertic.bytecoder.core.ir.AnalysisException, which also encapsulates the AnalysisStack. AnalysisException is caught and handled by the CLI, but not in the MavenMojo yet.

Reply:
fixed